app/controllers/coronavirus_form/data_export_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "csv"
+
+class CoronavirusForm::DataExportController < ApplicationController
+  def show
+    respond_to do |format|
+      format.html { render controller_path }
```

Comment:
Will this just be an empty page? Should we think about maybe having some kind of view?

Reply:
Yes, it will be a blank page.  Should we have some sort of content here?

Reply:
I could just remove the HTML line here.  Although we'd get raise an exception if users try navigating in their browser without the `.csv` suffix or correct accept header.

Reply:
An interesting thing happens here. If you accidentally go to `/data-export` without the `.csv` an empty page is rendered:

<img width="1532" alt="Screenshot 2020-04-16 at 11 07 17" src="https://user-images.githubusercontent.com/5793815/79445059-eed53a80-7fd3-11ea-821c-05872bc26e37.png">

I don't see this as a blocker, but it is interesting.

Reply:
I think an exception is better than a blank page. Or we could render an HTML page with a link to the `.csv` route?

Reply:
Turns out Rails will return `406 Not Acceptable` (https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406) if we don't specify anything to render here.  That feels like the correct response code, so I'll use the default behaviour.